shared/js/background/utils.es6.js

```diff
@@ -1,12 +1,35 @@
 import browser from 'webextension-polyfill'
-import { getExtensionVersion } from './wrapper.es6'
+import { getExtensionVersion, getFromSessionStorage, setToSessionStorage } from './wrapper.es6'
 import tdsStorage from './storage/tds.es6'
 import settings from './settings.es6'
 import load from './load.es6'
 import * as tldts from 'tldts'
 import parseUserAgentString from '../shared-utils/parse-user-agent-string.es6'
+import sha1 from '../shared-utils/sha1'
 const browserInfo = parseUserAgentString()
 
+/**
+ * Produce a random float, same output as Math.random()
```

Comment:
I think this comment should explain the purpose or appropriate use of this function a bit more. The reader might ask "If it's the same output as `Math.random()`, then why not just use `Math.random()`?"